competencySyncScript

```diff
@@ -28,11 +35,12 @@ function testRequests() {
       var fileName = file.getName(); //get the file name
       if (fileName != 'TEMPLATE') { //ignore files with name 'TEMPLATES'
         var indexOfNameInGithubFileNames = githubFileNames.indexOf(fileName);
-        
         //Checks if Drive file exists in GitHub repo (by file name)
         if (indexOfNameInGithubFileNames != -1) { //indexOf() returns -1 if value does not exist in array
```

Comment:
I would combine line 37 and the insides of the if conditional as a function so you can get rid of the comment
```
if (checkIfDriveFileExists(githubFilesNames, filename)) {
}
```